src/OneDrive.Sdk.Authentication.Desktop/Business/AuthenticationContextWrapper.cs

```diff
@@ -61,7 +61,7 @@ public ITokenCache TokenCache
         /// <param name="resource">The resource to authenticate against.</param>
         /// <param name="clientCredential">The client credential of the application.</param>
         /// <returns>The <see cref="IAuthenticationResult"/>.</returns>
-        public async Task<IAuthenticationResult> AcquireDaemonTokenSilentAsync(string resource, ClientCredential clientCredential)
+        public async Task<IAuthenticationResult> AcquireTokenSilentAsync(string resource, ClientCredential clientCredential)
```

Comment:
Yikes sorry, this should actually be `AcquireTokenAsync`. It should match the call to `AuthenticationContext` to make it a little easier for the client to figure out what's happening under the covers. Sorry if I gave you the wrong name before :(